README.md

```diff
@@ -1,248 +1,417 @@
+*or* "***Front End***" *of the stack*)
 
-## *Say No to Untested Software (Web/Apps)!*
+The "Stack" we use is _evolving_ with the industry _innovation_.
+Please see: https://github.com/dwyl/technology-stack/
+and if you want to `#GetInvolved` in the discussion,
+read and comment on the issues: https://github.com/dwyl/technology-stack/issues
 
-The days of poor quality software are *numbered*; people who continue to expect *inconsistent results* to be rewarded are in for a "*surprise*"!
+### You Will Learn "_In Demand_" Tech Tools/Skills
 
-![They Took Er Jerbs](https://raw.github.com/nelsonic/practical-js-tdd/master/manuscript/images/00-they-took-our-jerbs.jpg "They Took Er Jerbs")
+You will learn 3 of the "Top 10" most popular technologies of 2016
+(_which are still just as popular/relevant in 2017!_)
 
+https://stackoverflow.com/research/developer-survey-2016#technology
+![stackoverflow-most-popular-technologies](https://cloud.githubusercontent.com/assets/194400/21954032/a94f0128-da3e-11e6-95d4-64757a6175e7.png)
 
-This book is our effort to *accelerate* the process towards ending un-tested software!
-We invite *you* to be part of the ***TQM Software*** *movement* and bring all your friends/colleagues!
+_And_... we will _also_ cover a few of the most "_Wanted_" (_i.e. in demand!_)
+technology skills which are _dramatically_ improving people's productivity,
+effectiveness and quality of output!
 
- <br />
+https://stackoverflow.com/research/developer-survey-2016#most-popular-technologies-other
+![stackoverflow-other-technologies](https://cloud.githubusercontent.com/assets/194400/21954067/28e98638-da3f-11e6-9e2a-0b1b080a826a.png)
 
+## "*Full-Stack*" JavaScript?
 
-# *Alpha* (Book) Preview
+The term "*Full Stack*" Developer has been around for a while.
+We like [@laurencegellert](https://twitter.com/laurencegellert)'s
+explanation:
+
+> "*a Full Stack Developer is someone with* ***familiarity in each layer***, ***if not mastery in many*** *and a* ***genuine interest in all software technology***."
+
+We agree with [@peteryared](https://twitter.com/peteryared) that it's naive to expect *anyone* to be an "*expert*" in *every* aspect of our stack,
+but we do think that by using ***JavaScript*** as the programming language for the server (Node.js), client (*take your pick of framework!*) and datastore
+(*any of the NoSQL datastores which store JSON documents*), it's easier to go "full stack".
+
+See: http://techcrunch.com/2014/11/08/the-rise-and-fall-of-the-full-stack-developer/
+
+### "*Front End*"?
+
+There is ***one (programming) language*** that *all* web
+browsers "understand": ***JavaScript***.  
+As a result it is the _most popular programming language_
+(_see StackOverflow survey results above_!)
+
+Additionally you will need to know two other "languages": HTML and CSS ... but:
+
+![stay calm](https://cloud.githubusercontent.com/assets/194400/21954158/cea1b17a-da41-11e6-9f0e-a73627125934.png)
```

Comment:
Feels like it's missing something - "I wasn't thinking of panicking but now you're telling me not to... why shouldn't I?"

I was expecting to know why I shouldn't be worried in the sentence after the picture.